clients/admin-ui/src/features/common/PickerCard.tsx

```diff
@@ -58,9 +58,34 @@ export interface Props<T> {
   toggle?: ReactNode;
   onViewMore?: () => void;
   numSelected: number;
+  // eslint thinks this is unused because it's the prop type for two components
+  // and unused on one of them
+  // eslint-disable-next-line react/no-unused-prop-types
+  boldText?: boolean;
```

Comment:
The bold text/title case is used in the location/regulation picker but it just didn't look quite right in this context and I didn't want to reimplement all the checkbox list logic for a component that was like 95% of what I needed.  What would you suggest instead here?  (This step of the form is not in the designs-- using the "standard" checkbox list was a later scope cut.)

Reply:
Consistency between the 2 would definitely be ideal. Otherwise my suggestion in that case would be to create a `variant` property instead:
```variant: "emphasis" | "normal"```